<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comment_Like</class>
 <widget class="QWidget" name="Comment_Like">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Song_pic">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Song_name">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>260</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="Comment_section">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>1431</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1429</width>
      <height>349</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Like_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Like</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_comment">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>320</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Comment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="write_comment">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>1301</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1641</width>
     <height>861</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 47, 51);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_favorite">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Favorite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_playlist">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add To Playlist</string>
   </property>
  </widget>
  <widget class="QLabel" name="question_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Which Play List You want to add the song to?</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="playlists_scrollBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>289</width>
      <height>229</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Song_Artist">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>30</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Song_AgeCategory">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Song_Album">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>130</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Song_genr">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>180</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Song_Country">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="show_lyrics">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>240</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Lyrics</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Lyrics">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>20</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>Song_pic</zorder>
  <zorder>Song_name</zorder>
  <zorder>Comment_section</zorder>
  <zorder>Like_button</zorder>
  <zorder>Add_comment</zorder>
  <zorder>write_comment</zorder>
  <zorder>Back</zorder>
  <zorder>add_favorite</zorder>
  <zorder>add_playlist</zorder>
  <zorder>question_label</zorder>
  <zorder>playlists_scrollBar</zorder>
  <zorder>Song_Artist</zorder>
  <zorder>Song_AgeCategory</zorder>
  <zorder>Song_Album</zorder>
  <zorder>Song_genr</zorder>
  <zorder>Song_Country</zorder>
  <zorder>show_lyrics</zorder>
  <zorder>textEdit_Lyrics</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
